Assets/UI/Scripts/TabWidget.gd

```diff
@@ -17,3 +18,24 @@ func _process(_delta: float) -> void:
 func _on_LogBookButton_pressed():
 	get_tree().root.add_child(log_book.instance())
 	Audio.play_snd_click()
+	
+func _change_on_select(selected_units):
+	if selected_units.size() == 1:
+		var selected_unit = selected_units[0]
+		# only one Unit, so probably only possible to build etc. if only one Unit selected
+		if selected_unit.get_groups().find("ships") == 1: # Its a ship
+			_set_detail_body(ship_menu)
+			ship_menu.set_selected_ship(selected_unit)
+			_set_detail_visibility(true)			
+		if selected_units[0].get_groups().find("ships") != 1: # Its not a ship. Do something else
+			pass
+	elif selected_units.size() > 1: # multiple units selected, do something else
+		pass
+	pass
```

Comment:
Remove unused `pass`, and consider removing (or commenting out) blocks which only contain `pass`.